transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -188,7 +188,11 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("linux")) {
             throw new IllegalStateException("Only supported on Linux");
         }
-        NativeLibraryLoader.load("netty-transport-native-epoll", PlatformDependent.getClassLoader(Native.class));
+        String []libraryNames = new String[]{
+          "netty-transport-native-epoll",
+          "netty_transport_native_epoll"
```

Comment:
yes, since the name of the so library extracted has  dashes in it.  If you want to change that name too, it would be possible.